### ISSUE TITLE
Seems to fail the debian-specific tests on RedHat-ish distributions

### ISSUE THREAD

**Issue by xtaran** [xtaran/run-parts#1]
`t/basic-functionality.t` seems to fail on RedHat-ish distributions due `/bin/run-parts` generating different output (prefixing the script being run) than on Debian and derivatives.
- http://www.cpantesters.org/cpan/report/3a648cb4-391e-11e5-91e5-762cb55f639b
- http://www.cpantesters.org/cpan/report/1b1f726e-386b-11e5-b029-8929b55f639b

Despite both reports were with Perl 5.23.1 I doubt that this issue is dependent on the Perl version being used, at least after a first glance.

**Comment by xtaran** [comment]
This seems to be the same issue as https://rt.cpan.org/Public/Bug/Display.html?id=127691 by @srezic. Details of his bug report:

# Test suite fails on CentOS7 and Fedora 28

```
#   Failed test 'Returns output of ran executables (debian)'
#   at t/basic-functionality.t line 81.
# +---+------------------------+---+-------------+
# | Ln|Got                     | Ln|Expected     |
# +---+------------------------+---+-------------+
# *  1|'t/basic-dummy/script:  *  1|'Works       *
# *  2|\n                      *   |             |
# *  3|Works                   *   |             |
# *  4|t/basic-dummy/script2:  *   |             |
# *  5|\n                      *   |             |
# |  6|Works, too!             |  2|Works, too!  |
# |  7|'                       |  3|'            |
# +---+------------------------+---+-------------+

#   Failed test 'Returns output of ran executables (run-parts)'
#   at t/basic-functionality.t line 81.
# +---+------------------------+---+-------------+
# | Ln|Got                     | Ln|Expected     |
# +---+------------------------+---+-------------+
# *  1|'t/basic-dummy/script:  *  1|'Works       *
# *  2|\n                      *   |             |
# *  3|Works                   *   |             |
# *  4|t/basic-dummy/script2:  *   |             |
# *  5|\n                      *   |             |
# |  6|Works, too!             |  2|Works, too!  |
# |  7|'                       |  3|'            |
# +---+------------------------+---+-------------+

#   Failed test 'Returns output of ran executables (default)'
#   at t/basic-functionality.t line 81.
# +---+------------------------+---+-------------+
# | Ln|Got                     | Ln|Expected     |
# +---+------------------------+---+-------------+
# *  1|'t/basic-dummy/script:  *  1|'Works       *
# *  2|\n                      *   |             |
# *  3|Works                   *   |             |
# *  4|t/basic-dummy/script2:  *   |             |
# *  5|\n                      *   |             |
# |  6|Works, too!             |  2|Works, too!  |
# |  7|'                       |  3|'            |
# +---+------------------------+---+-------------+
# Looks like you failed 3 tests of 51.
t/basic-functionality.t .... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/51 subtests 
t/boilerplate.t ............ ok

#   Failed test 'Both return same output of ran executables'
#   at t/compare-functionality.t line 45.
# +---+------------------------+---+-------------+
# | Ln|Got                     | Ln|Expected     |
# +---+------------------------+---+-------------+
# *  1|'t/basic-dummy/script:  *  1|'Works       *
# *  2|\n                      *   |             |
# *  3|Works                   *   |             |
# *  4|t/basic-dummy/script2:  *   |             |
# *  5|\n                      *   |             |
# |  6|Works, too!             |  2|Works, too!  |
# |  7|'                       |  3|'            |
# +---+------------------------+---+-------------+
# Looks like you failed 1 test of 8.
t/compare-functionality.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/8 subtests 
```

**Comment by xtaran** [comment]
Fixed with 3a07a67ec08907d6585a6e3eaf52aba28b4c1761
